frontend_lib/src/tracimLiveMessage.js

```diff
@@ -17,6 +11,9 @@ const CREATED = 'created'
 const MODIFIED = 'modified'
 const DELETED = 'deleted'
 
+export const TLM_SUB_TYPE = {
```

Comment:
Personally, I used CONTENT_TYPE to make event names.
But I think your way is cleaner.
If we keep your way, we will have 2 ways of declaring TLM_SUB_TYPE which I don't like.
So, refactor Dashboard.jsx component in frontend to use your constant instead of CONTENT_TYPE